app/src/main/java/com/going/doorip/di/AuthInterceptor.kt

```diff
@@ -50,18 +58,17 @@ class AuthInterceptor @Inject constructor(
                                 .build()
 
                         return chain.proceed(newRequest)
-                    }.onFailure {
-                        dataStore.clearInfo()
                     }
+                } catch (t: Throwable) { }
+                dataStore.clearInfo()
 
-                    dataStore.clearInfo()
-
-                    // refreshToken 만료 처리를 위한 리프레시 토큰 만료 코드 포함 리스폰스 리턴
-                    return response
-                } catch (t: Throwable) {
-                    dataStore.clearInfo()
-
-                    Timber.e(t)
+                Handler(Looper.getMainLooper()).post {
+                    context.toast(TOKEN_EXPIRED_ERROR)
+                    context.startActivity(
+                        Intent(context, SignInActivity::class.java).apply {
+                            addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK)
+                        },
+                    )
```

Comment:
```suggestion
                        Intent(context, SignInActivity::class.java).apply {
                            addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK)
                            context.startActivity(this)
                        },
                    )
```
형태가 좀 더 인텐트 중심의 코드여서 이쁜 것 같아요 !

Reply:
이게 훨씬 좋네요! 수정했습니당~!